proposals/0245-sharing-wifi-ssid-and-password.md

```diff
@@ -637,46 +354,60 @@ Once network parameters are established with the first device, `OnNetworkConfigu
+A vehicle should be able to share credentials of the external hotspot only if `allowAccessPointSharing` is set to true in mobile application `NetworkCapabilities` and the ini file.
 
 If mobile is an access point and `allowAccessPointSharing = false`, vehicle is not able to establish Wifi connection with other devices.
 
-In that case, HMI should provide SDL `OnSystemCapabilityUpdate` with appropriate `hostingWiFiSupported`, and `autoJoinWiFiSupported`.
+In that case, HMI should provide SDL `OnSystemCapabilityUpdate` with appropriate `canHostWiFiNetwork`, and `autoJoinWiFiSupported`.
 
-Once vehicle is connected to the mobile hotspot, HMI should send `OnSystemCapabilityUpdate` with `hostingWiFiSupported = true` and `preferredNetworkHost = EXTERNAL`. If the new application is compatible with HMI network capabilities, Core will send `OnNetworkConfiguration` with `networkHost = EXTERNAL` to HMI and new mobile application. Then HMI should send `JoinNetwork` with the credentials of the external hotspot. After the new application receives this request, it is responsible for its rejection or connection to the external hotspot.
+Once vehicle is connected to the mobile hotspot, HMI should send `OnSystemCapabilityUpdate` with `canHostWiFiNetwork = true` and `preferredNetworkHost = VEHICLE`. If the new application is compatible with HMI network capabilities, Core will send `OnSystemCapabilityUpdate` with `networkHost = VEHICLE` to HMI and new mobile application. Then HMI should send `JoinNetwork` with the credentials of the external hotspot.
 
+<!--
+@startuml ../assets/proposals/0245-sharing-wifi-ssid-and-password/mobile_access_point_multidev_externalhost
+!include ../assets/proposals/0245-sharing-wifi-ssid-and-password/mobile_access_point_multidev_externalhost.puml
+@enduml
+-->
 ![Vehicle access point multiple devices](../assets/proposals/0245-sharing-wifi-ssid-and-password/mobile_access_point_multidev_externalhost.png)
 
-
 ### RPCs Encryption
 
-WiFi credentials are considered sensitive data, and should not be shared without encryption.
+WiFi credentials are considered as sensitive data, and should not be shared without encryption.
 [SDL 0207 - RPC message protection](https://github.com/smartdevicelink/sdl_evolution/issues/634) feature.
 
 RPC message protection is controlled completely by Policy Table.
 
 It is advised to restrict `JoinNetwork` to be sent only by encrypted RPC service, but it is up to particular OEM. SDL does not enforce `JoinNetwork` to be sent only by encrypted RPC service.
 
-If `JoinNetwork` or `OnNetworkConfiguration` should not be protected, there are no additional issues.
-If `JoinNetwork` or `OnNetworkConfiguration` are allowed only via protected RPC service, the following rules are applied:
+If `JoinNetwork` or `OnSystemCapabilityUpdate` should not be protected, there are no additional issues.

```

Comment:
```suggestion
If `JoinNetwork` or `OnSystemCapabilityUpdated` should not be protected, there are no additional issues.
```
Please update all references to `OnSystemCapabilityUpdate` to be `OnSystemCapabilityUpdated`